src/orion/core/cli/checks/creation.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.core.cli.checks.creation` -- Creation stage for database checks
+===========================================================================
+
+.. module:: creation
+    :platform: Unix
+    :synopsis: Checks for the creation of a `Database` object.
+
+"""
+
+from orion.core.io.database import Database
+from orion.core.utils.decorators import register_check
+from orion.core.utils.exceptions import CheckError
+
+
+class _Checks:
+    checks = []
+
+
+class CreationStage:
+    """The creation stage of the checks."""
+
+    def __init__(self, presence_stage):
+        """Create an intance of the stage.
+
+        Parameters
+        ----------
+        presence_stage: `PresenceStage`
+            An instance of the previous stage.
+
+        """
+        self.p_stage = presence_stage
+        self.instance = None
+
+    @staticmethod
+    def checks():
+        """Return checklist."""
+        return _Checks.checks
+
+    @register_check(_Checks.checks, "Check if database of specified type can be created... ")
```

Comment:
Why do you use a decorator for this? Why not just print it at the beginning of the method call?